lab_1_keywords_tfidf/main.py

```diff
@@ -118,6 +200,38 @@ def calculate_expected_frequency(
 
     In case of corrupt input arguments, None is returned
     """
+    if isinstance(doc_freqs, dict) is False or isinstance(corpus_freqs, dict) is False:
+        return None
+    else:
+        for k, v in doc_freqs.items():
+            if isinstance(k, str) is False or isinstance(v, int) is False:
+                return None
+        for k, v in corpus_freqs.items():
+            if isinstance(k, str) is False or isinstance(v, int) is False:
+                return None
+        # подумать как решить это по другому
+        words_in_doc = 0
+        words_in_col = 0
+        doc_freqs_val = list(doc_freqs.values())
+        col_freqs_val = list(doc_freqs.values())
```

Comment:
did you put `doc_freqs` in both lines here intentionally? you could want corpus_freqs for the col_freqs_val